.buildkite/scripts/grapl_testing_stack/dump_artifacts.sh

```diff
@@ -17,5 +17,5 @@ readonly _NOMAD_ADDRESS
 
 cd "${REPOSITORY_ROOT}"
 NOMAD_ADDRESS="${_NOMAD_ADDRESS}" ./pants run \
-    ./etc/ci_scripts/dump_artifacts -- \
+    ./etc/ci_scripts/dump_artifacts:dump_artifacts -- \
```

Comment:
It would be better to use our alias: https://github.com/grapl-security/grapl/blob/3b8e01a2dfc10c68103d969371025e4e8aee7c8a/pants.toml#L239

This would then just be `./pants dump-artifacts --no-dump-agent-logs`

(I was wracking my brain trying to figure out how this made it through, when I was sure I'd updated this command for the new Pants syntax... guess I missed this invocation.)